initrd/etc/functions

```diff
@@ -30,13 +37,30 @@ DEBUG() {
 	fi
 }
 
+# Use TRACE to trace control flow through Heads.  This is usually called by
+# TRACE_FUNC, but you can use it to additionally trace parameter values, etc.
+# Usually, use this to display unprocessed parameters that your script or
+# function received.  For information about the logic occurring in your script
+# or function, use DEBUG.  See doc/logging.md.
 TRACE() {
 	if [ "$CONFIG_ENABLE_FUNCTION_TRACING_OUTPUT" = "y" ]; then
 		echo "TRACE: $*" | tee -a /tmp/debug.log /dev/kmsg >/dev/null
 	fi
 }
 
-# Function to manage information output level to the console/debug.log
+# Use NOTE to explain behaviors that are _likely_ to be unexpected or confusing.
+# Unlike INFO, this cannot be hidden, as the explained behavior would be too
+# confusing without this output.
+# Don't overuse this - too much NOTE output will cause users to ignore it.  See
+# doc/logging.md.
+NOTE() {
+	echo "NOTE:" "$@" | tee -a /tmp/debug.log
+}
```

Comment:
@JonathonHall-Purism I love eveyrhing I see here, but the change to warn to note removing the 2 seconds that was added by warn. Not a big blocker, but sometimes, the note happening before a clear, a reboot, or a binary input might just pass as silence.

Your call, but its also part of why warn was used (the sleep) permitting user to acknoeldge in absence of output and action on the console.